src/libraries/System.Private.CoreLib/src/System/Buffers/Text/Utf8Formatter/Utf8Formatter.Date.O.cs

```diff
@@ -60,7 +60,7 @@ private static bool TryFormatDateTimeO(DateTime value, TimeSpan offset, Span<byt
             destination[7] = Utf8Constants.Minus;
 
             FormattingHelpers.WriteTwoDecimalDigits((uint)day, destination, 8);
-            destination[10] = TimeMarker;
+            destination[10] = (byte)'T';
```

Comment:
Why does it make it less readable?  This is formatting the specific format outlined in the comment at the top of the method.  With this change, I can see exactly what character it's writing.  Without it, I have to look to see how TimeMarker is defined to know what it is.  That said, I removed it because I was removing every other constant/array that was defined, and it didn't make sense to me to leave just these T and Z constants.